metron-extensions/metron-parser-extensions/metron-parser-bro-extension/metron-parser-bro/src/main/config/zookeeper/enrichments/bro.json

```diff
@@ -0,0 +1,17 @@
+{
+    "enrichment" : {
```

Comment:
Ok - so remove enrichment from the archetype - and leave it a UI issue ( where they can create it I think ). 

Removing it from bro, yaf and snort however will be changing prior behavior, since they run kibana etc.  I think they need to stay or it is a regression for that part of the system.